draft-zip.md

```diff
@@ -35,6 +35,8 @@ This ZIP attempts to establish a symbiotic relationship between miner incentives
 
 Implementing this change allows the ZSF to accrue value earlier. By ensuring a consistent source of funding, the ZSF contributes to bolstering the Zcash network’s long-term security and sustainability.
 
+This ZIP depends on the ZIP introducing the Zcash Sustainability Fund (ZIP-XXX).
```

Comment:
This doesn't quite capture the different dependencies here, it's subtle but important.

```suggestion
This ZIP depends on the ZIP introducing the Zcash Sustainability Fund (ZIP-XXX). The fees added to the ZSF wont be used until there are payouts from the ZSF. This could happen via subsidy smoothing (ZIP-YYY), or some other mechanism.
```